src/internal/modules/billing/routes/create-bill-run.js

```diff
@@ -83,6 +83,21 @@ module.exports = {
     }
   },
 
+  getBillingBatchSroc: {
+    method: 'GET',
+    path: '/billing/batch/sroc/{region}',
+    handler: controller.getBillingBatchSroc,
+    config: {
+      auth: { scope: allowedScopes },
+      description: 'post handler for creating an sroc billing batch',
```

Comment:
I found something to comment on!

```suggestion
      description: 'get handler for creating an sroc billing batch',
```